installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,76 @@
+
+See the xref:../../architecture/architecture-installation.html#installation-process_architecture-installation[Installation process] for more information about installer-provisioned and user-provisioned installation processes.
+
+=== Installer-provisioned infrastructure installation of {product-title} on vSphere
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[Installing a cluster on vSphere]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with no customization.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc#installing-vsphere-installer-provisioned-customizations[Installing a cluster on vSphere with customizations]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with the default customization options.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installing-vsphere-installer-provisioned-network-customizations[Installing a cluster on vSphere with network customizations]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with network customization options. The installation program allows for some customization to be applied at the installation stage.
+
+* **xref:../../installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc#installing-restricted-networks-installer-provisioned-vsphere[Installing a cluster on vSphere in a restricted network]**: You can install a cluster on VMware vSphere infrastructure in a restricted network by creating an internal mirror of the installation release content.
+ You can use this method to deploy {product-title} on an internal network that is not visible to the Internet.
+
+=== User-provisioned infrastructure installation of {product-title} on vSphere
+
+* **xref:../../installing/installing_vsphere/installing-vsphere.adoc#[Installing a cluster on vSphere]**: You can install {product-title} on VMware vSphere infrastructure that you provision.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-network-customizations.adoc#installing-vsphere-network-customizations[Installing a cluster on vSphere with network customizations]**: You can install {product-title} on VMware vSphere infrastructure that you provision with customized network configuration options.
+
+* **xref:../../installing/installing_vsphere/installing-restricted-networks-vsphere.adoc#installing-restricted-networks-vsphere[Installing a cluster on vSphere in a restricted network]**: {product-title} can be installed on VMware vSphere infrastructure that you provision in a restricted network.
```

Comment:
You could add "with user-provisioned infrastructure" to the end of the link title in this bullet point, to differentiate it from the bullet that has the same heading in the IPI section. The assembly that it links to is "Installing a cluster on vSphere in a restricted network with user-provisioned infrastructure".